exporter/awsemfexporter/metric_translator.go

```diff
@@ -116,12 +124,73 @@ func TranslateOtToCWMetric(rm *pdata.ResourceMetrics, dimensionRollupOption stri
 				continue
 			}
 			cwMetricList := getMeasurements(&metric, namespace, OTLib, dimensionRollupOption)
+			batchCWMetrics(cwMetricList, groupedCWMetricMap, cwMetricsMap)
 			cwMetricLists = append(cwMetricLists, cwMetricList...)
 		}
 	}
 	return cwMetricLists, totalDroppedMetrics
 }
 
+func batchCWMetrics(cwMetricList []*CWMetrics, groupedCWMetricMap map[string]*GroupedCWMetric, cwMetricsMap map[string]*CWMetrics) {
+	for _, i := range cwMetricList {
+		if len(i.Measurements[0].Dimensions[0]) > 0 {
+			var key = strings.Join(i.Measurements[0].Dimensions[0],"")
```

Comment:
is there a reason for the use of `var` here? If not, the standard is to use `:=` 
```suggestion
			key := strings.Join(i.Measurements[0].Dimensions[0], "")
```